diff --git a/base.mk b/base.mk
--- a/base.mk
+++ b/base.mk
@@ -81,8 +81,6 @@ PRODUCT_COPY_FILES += \
 	$(call find-copy-subdir-files,*,device/phh/treble/huawei_charger/files,system/etc/charger)
 
 PRODUCT_COPY_FILES += \
-	device/phh/treble/twrp/twrp.rc:system/etc/init/twrp.rc \
-	device/phh/treble/twrp/twrp.sh:system/bin/twrp.sh \
 	device/phh/treble/twrp/busybox-armv7l:system/bin/busybox_phh
 
 PRODUCT_PACKAGES += \
